source/home-energy-management/index.html

```diff
@@ -51,15 +57,19 @@
 
 <div class="material-card text banner-overlay">
     <div>
-        <div class="banner-overlay-header">Don't have Home Assistant? It's free.</div>
+        <div class="banner-overlay-header">Home Energy Management</div>
         <div class="banner-overlay-content">
-        With Home Assistant we want to do our part to help take on the climate crisis, and help you do the same. One part of fighting the climate crisis is making sure our homes are energy efficient and use low-carbon energy sources.
-        <br />Get started today!
+          With Home Assistant we want to do our part to help take on the climate crisis, and help you do the same. One part of fighting the climate crisis is making sure our homes are energy efficient and use low-carbon energy sources.
+          <br><br>
+          Home Assistant helps you to monitor your energy usage, transition to sustainable energy and save money.
         </div>
     </div>
-    <a href="/getting-started">
-        <div class="banner-overlay-button bottom right">
-            <div>Get started</div>
-        </div>
-    </a>
+    <div class="links">
+      <a href="/getting-started">
+        Get started
+      </a>
+      <a href="https://demo.home-assistant.io/#/energy">
```

Comment:
```suggestion
      <a href="https://demo.home-assistant.io/#/energy" target="_blank">
```
When we navigate out of the site, we should do so in a new tab to not lose the viewer